sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadClientTest.java

```diff
@@ -361,6 +364,7 @@ public void canUpdateExistingMessageWithResponse(HttpClient httpClient) {
 
         ChatMessage message = chatThreadClient.getMessage(response.getId());
         assertEquals(message.getContent().getMessage(), updateMessageRequest.getContent());
+        assertTrue(message.getMetadata().equals(updateMessageRequest.getMetadata()));
     }
 
     @ParameterizedTest
```

Comment:
Maybe add new test case to update metadata to prove expected behavior. New ones are added. Same named ones updated. Old ones removed if that's what expected to happen

Reply:
I will update this test case to reflect the different behaviors